src/pages/sighting/EditSightingMetadata.jsx

```diff
@@ -28,14 +29,27 @@ export default function EditSightingMetadata({
   metadata,
   onClose,
   refreshSightingData,
+  pending,
 }) {
   const {
-    updateProperties,
-    loading,
-    error,
-    setError,
+    updateProperties: updateSightingProperties,
+    loading: sightingLoading,
+    error: sightingError,
+    setError: setSightingError,
   } = usePatchSighting();
 
+  const {
+    updateProperties: updateAgsProperties,
+    loading: agsLoading,
+    error: agsError,
+    setError: setAgsError,
+  } = usePatchAssetGroupSighting();
+
+  const error = pending ? agsError : sightingError;
```

Comment:
It's not clear to me how pending modulates which of these gets assigned to error.

Reply:
Same for the three lines below this.

Reply:
oh, if an AGS is being viewed, then `updateAgsProperties` will always be used instead of `updateSightingProperties`. and so on